specification/electronic-prescription-service-api.yaml

```diff
@@ -681,6 +700,9 @@ paths:
         required: true
         content:
           application/fhir+json:
+            schema:
+              type: object
+              description: The request to withdraw a dispense notification, expressed as a [Task FHIR resource](https://simplifier.net/resolve?target=simplifier&scope=uk.nhsdigital.r4&canonical=https://fhir.nhs.uk/StructureDefinition/NHSDigital-Task).
```

Comment:
same as above

Reply:
Fixed. Also changed similar link in the endpoint description.